### ISSUE TITLE
Fill image after rotate

### ISSUE THREAD

**Issue by annebione** [advanced-cropper/react-advanced-cropper#69]
Hello,

I'm having an issue in figuring out how to make the cropper area fill the entire image after rotation. As per the docs, I've used the `defaultSize` prop to great results on initial image load, as you can see:

![image](https://github.com/user-attachments/assets/a75ee516-0cae-49b3-bc1a-9104ce30aa52)

However, after rotation, the cropper area keeps the original ratio, despite the image w/h changing drastically. 

![image](https://github.com/user-attachments/assets/75dfdf73-ca5a-41a0-b020-da55d7777a75)

TL;DR: I'd like to figure out a way to make the cropper area to fill the entire image size despite rotation.

I understand the `transformImageAlgorithm` prop passes the `state` object to the algorithm where I could find the current image proportions, but I'm having trouble understanding how I could change the cropper on rotate.
